### PR TITLE
Update DG and UG

### PR DESCRIPTION
**Pull request by TianYong-Goh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Wishful Shrinking is a **desktop app for managing your diet, keeping track of yo
 
    * **`recipe`** : Lists all recipes.
 
-   * **`addR`**` n/salad i/lettuce, carrots, olive oil` : Adds a `salad` recipe to Wishful Shrinking.
+   * **`addR`**` n/salad i/lettuce, carrots, olive oil c/40` : Adds a `salad` recipe to Wishful Shrinking.
```
Comment (caitlinjee) [inline docs/UserGuide.md:30]: should we write the whole proper command?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -228,15 +247,16 @@ _{give a remark to the recipe}_
...
-**Add recipe** | `addR n/TITLE i/INGREDIENTS` <br> e.g., `addR n/salad i/lettuce, carrots, olive oil`
+**Add recipe** | `addR n/TITLE i/INGREDIENT[, MORE INGREDIENT] c/CALORIES img/IMAGE inst/INSTRUCTION... [t/TAG]...` <br> e.g., `addR n/salad i/lettuce, carrots, olive oil c/80`
 **Add Ingredient to the fridge** | `addF i/INGREDIENTS` <br> e.g., `addF i/banana, green peas, salmon fish`
 **Delete recipe** | `deleteR INDEX`<br> e.g., `deleteR 3`
 **Delete Ingredient from the fridge** | `deleteF INDEX`<br> e.g., `deleteF 3`
 **Delete recipe eaten**| `deleteC INDEX` <br> e.g., `deleteC 3`
-**Search recipe** | `searchR i/INGREDIENT` OR `searchR n/TITLE` OR `searchR t/TAG` <br> e.g., `searchR i/lettuce` `searchR n/salad` `searchR t/healthy`
+**Search recipe** | `searchR i/INGREDIENT` OR `searchR n/TITLE` OR `searchR t/TAG` <br> e.g. `searchR i/lettuce`, `searchR n/salad`, `searchR t/healthy`
 **Search Ingredient in the fridge** | `searchF KEYWORD`<br> e.g., `searchF avocado`
 **List recipe** | `recipe`
```
Comment (caitlinjee) [inline docs/UserGuide.md]: dont forget to change recipe to recipes

File: docs/UserGuide.md [hunk 3]
```diff
@@ -228,15 +247,16 @@ _{give a remark to the recipe}_
 
 Features | Format, Examples
 --------|------------------
-**Add recipe** | `addR n/TITLE i/INGREDIENTS` <br> e.g., `addR n/salad i/lettuce, carrots, olive oil`
+**Add recipe** | `addR n/TITLE i/INGREDIENT[, MORE INGREDIENT] c/CALORIES img/IMAGE inst/INSTRUCTION... [t/TAG]...` <br> e.g., `addR n/salad i/lettuce, carrots, olive oil c/80`
```
Comment (caitlinjee) [inline docs/UserGuide.md:250]: same as previous, full command?

**Review by caitlinjee — APPROVED** [review]
minor changes

**Review by daongochieu2810 — APPROVED** [review]

**Review by daongochieu2810 — APPROVED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (TianYong-Goh) [reply docs/UserGuide.md:30]: Jia qi changes the way of adding instruction and I not sure how does it look like as that part is not merged to the master yet. The addRecipeCommand UG documentation is base on what I know so far.

File: docs/UserGuide.md (hunk 2 above)
Comment (TianYong-Goh) [reply docs/UserGuide.md]: Ok changed. Will push later.

File: docs/UserGuide.md (hunk 3 above)
Comment (TianYong-Goh) [reply docs/UserGuide.md:250]: Same as the previous reply.

**Review by jiaqi20 — APPROVED** [review]
lgtm

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -149,6 +149,100 @@ The following sequence diagram shows how the undo operation works:
...
+#### Implementation
+Substitutability is used in Command and Parser:
+* `EatRecipeCommand` extends `Command`
+* `EatRecipeCommandParser` implements `Parser<EatRecipeCommand>`
+
+Given bellow is the simplified step on how eat recipe is done:
+
+Step 1:
+
+User input is change into command.
```
Comment (oliviajohansen) [inline docs/DeveloperGuide.md:166]: changes or is changed depending on whether we choose present/ past tense

**Review by oliviajohansen — APPROVED** [review]
lgtm

**Pull request merged**